changelog.txt

```diff
@@ -1,4 +1,233 @@
 == Changelog ==
+= 3.9.3 =
+Release date: Mar 9th, 2016
```

Comment:
In the existing changelog entries, we use the long month format (`March` instead of `Mar`). We should do the same to be consistent.